docs/network/index.md

```diff
@@ -28,9 +27,8 @@ Overview of Celo Mainnet, Alfajores L2 Testnet and Baklava Testnet.
 | Chain ID                   | 44787                                                                                                                                                                                                                                                                                                                                                                                                                                                                                   |
-| RPC Nodes                  | <ul><li>Open RPC Endpoint (Forno): [https://alfajores-forno.celo-testnet.org/](https://alfajores-forno.celo-testnet.org/)</li><li>Infura: [https://www.infura.io/networks/celo](https://www.infura.io/networks/celo)</li></ul> |
-| RPC Endpoint (best effort) | [https://alfajores-forno.celo-testnet.org/](https://alfajores-forno.celo-testnet.org/)                                                                                                                                                                                                                                                                                                                                                                                                  |
-| Block Explorer             | <ul><li>[https://celo-alfajores.blockscout.com/](https://celo-alfajores.blockscout.com/)</li><li>[https://alfajores.celoscan.io/](https://alfajores.celoscan.io/)</li></ul>                                                                                                                                                                                                                                                                                                            |
+| RPC Endpoint (best effort) | [https://alfajores-forno.celo-testnet.org](https://alfajores-forno.celo-testnet.org)                                                                                                                                                                                                                                                                                                                                                                                                  |
+| RPC Nodes                  | <ul><li>Ethereum JSON-RPC endpoint: [https://alfajores-forno.celo-testnet.org](https://alfajores-forno.celo-testnet.org) (op-geth kind)</li><li>OP RPC endpoint: [https://op.alfajores.celo-testnet.org/](https://op.alfajores.celo-testnet.org/), (op-node kind)OP RPC endpoint: [https://op.alfajores.celo-testnet.org/](https://op.alfajores.celo-testnet.org/) (op-node kind)</li><li>Infura: [https://www.infura.io/networks/celo](https://www.infura.io/networks/celo)</li></ul> |                                                                                                                                |
```

Comment:
I wouldn't mention the OP RPC endpoints here, I think it confuses people more that it helps them and few people need them.